Circumvent an attempt to run cross-compiled executables during configuration.

--- ./meson.build
+++ ./meson.build
@@ -215,6 +215,10 @@
   glade_datadir,
 )
 
+if meson.is_cross_build()
+  # On cross, the test program can't be run, so just assume tls_backend_support
+  tls_backend_support = 'true'
+else
 tls_backend_support_src = '''
 #include <stdio.h>
 #include <gio/gio.h>
@@ -225,6 +229,7 @@
 }
 '''
 tls_backend_support = cc.run(tls_backend_support_src, dependencies: gio_dep).stdout()
+endif
 
 output = '\nConfiguration:\n\n'
 output += '\tCompiler:                ' + cc.get_id() + '\n'
